### DISCUSSION TITLE
Audio glitches on refresh / switch page

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#437] [Tips]
Disable `Browser on RPi` in Features might cause audio glitches on refresh / switch page
- After system upgrade `pacman -Syu` which kernel `linux-raspberrypi` also upgraded.
- ~~On 64bit version - Very likely, kernel upgraded or not.~~
- Enable `Browser on RPi` ~~> Disable~~ might help.

**Comment by R101G** [comment]
Thanks for the warning, rern. I reported something similar under #343, but the problem disappeared, even with the local browser disabled.

I use the 64 bit image, and wonder why we still need 32 bit images for the 64 bit Pi's?
